modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaScreenContributorClassCheck.testjava

```diff
@@ -0,0 +1,52 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+@Component(service = PortalSettingsConfigurationScreenContributor.class)
+public class UpgradeJavaScreenContributorClassCheck
+	implements PortalSettingsConfigurationScreenContributor {
+
+	@Override
+	public String getCategoryKey() {
+		return "scim-name";
```

Comment:
@nicolas-sss, what is `scim`?

Can you please replace this term with something more meaningful from a test perspective?